aws/table_aws_config_configuration_recorder.go

```diff
@@ -30,6 +30,13 @@ func tableAwsConfigConfigurationRecorder(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_STRING,
 				Transform:   transform.FromField("Name"),
 			},
+			{
+				Name:        "arn",
+				Description: "Amazon resource name of Configuration Recorder",
```

Comment:
Is this comment consistent with the other `arn` columns? It looks a bit different.

Also, please note that all descriptions should end with periods. Please review description standards here - https://steampipe.io/docs/develop/standards#table-and-column-descriptions

Reply:
Updated the description as per our standard